### PR TITLE
Createcandidate

### PR DESCRIPTION
**Pull request by iturnmyscreenon**
created a candidate account creation page

### CODE REVIEW

File: frontend/src/components/Candidate.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export default function Candidate() {
```
Comment (Justine-Merlin) [inline frontend/src/components/Candidate.jsx:1]: 'Candidate' doesn't reflect what the component is. Maybe 'NewCandidateForm' ?

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

File: frontend/src/components/Candidate.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <form>
+      <p>Join Externatic as candidate</p>
+      <div>
```
[the remark this answers is not in the document]
Comment (iturnmyscreenon) [reply frontend/src/components/Candidate.jsx:5]: Deleted it in candidate.jsx and compagny.jsx too

File: frontend/src/components/Candidate.jsx (hunk 1 above)
Comment (iturnmyscreenon) [reply frontend/src/components/Candidate.jsx:1]: renamed the component name in NewCandidateForm
and compagny.jsx is now NewConsultantForm

File: frontend/src/components/Candidate.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          <label htmlFor="password">Mot de passe</label>
+          <input id="password" type="password" />
+        </div>
+        <div>
+          <label htmlFor="passwordConfirm">Comfirmer mot de passe</label>
+          <input id="passwordConfirm" type="password" />
+        </div>
+      </div>
+
+      <button type="button">Creer un compte</button>
```
[the remark this answers is not in the document]
Comment (iturnmyscreenon) [reply frontend/src/components/Candidate.jsx:28]: changed type of submit in NewCandidateForm and NewConsultantForm

**Pull request merged**